### ISSUE TITLE
Request: Add button to toolbar for enabling/disabling SSL Proxying

### ISSUE THREAD

**Issue by K1mVZ** [ProxymanApp/Proxyman#1147]
Sometimes I need to switch between an inhouse build and store build to compare behaviour. In the last build SSL Proxying Tool must be disabled because otherwise the calls are blocked. 

It would be useful to have a button in the toolbar to quickly enabling or disabling SSL Proxying (Tool) instead of going to Tools > SSL Proxying and check on/off 'Enable SSL Proxying Tool'. 

**Proxyman Version:** 3.1.0 (30100)
**MacOS Version:** Monterey 12.0.1

**Comment by NghiaTranUIT** [comment]
You're right. The custom Toolbar doesn't have the SSL Proxying List toggle.

I'm going to add it and send you a beta build soon 👍 

<img width="1337" alt="Screen Shot 2022-02-14 at 17 20 14" src="https://user-images.githubusercontent.com/5878421/153845704-52d4d9f3-9eb3-4b09-8a8f-38bb54aa6478.png">

**Comment by NghiaTranUIT** [comment]
@K1mVZ please try this beta build: https://proxyman.s3.us-east-2.amazonaws.com/beta/Proxyman_3.1.0_Add_SSL_Proxying_to_Toolbar_&_System_Menu.dmg

From now, you can add SSL Proxying to your toolbar or Toggle it from the System Menu bar 👍 

<img width="1404" alt="Screen_Shot_2022-02-20_at_09_35_49" src="https://user-images.githubusercontent.com/5878421/154826399-3e9e4e0e-d9ab-443f-b3c6-4b8016f89b55.png">

**Comment by K1mVZ** [comment]
@NghiaTranUIT works like a charm! Thanks!
